source/administration/object-management/object-lifecycle-management.rst

```diff
@@ -58,6 +58,13 @@ The output includes the storage tier of each object:
 - ``STANDARD`` marks objects stored on the MinIO deployment.
 - ``WARM`` marks objects stored on the remote tier with matching name.
 
+.. important::
+
+   MinIO Object Transition does **not** provide backup and recovery functionality.
```

Comment:
I think we need to add something about when **to** use object transition.
Maybe something like:

```suggestion
   MinIO Object Transition is intended to move older and/or less frequently used objects from faster and typically more expensive storage devices to slower, typically less expensive options.
   Object Transition does **not** provide backup and recovery functionality.
```

Maybe something like the first line further up in the page rather than as part of this admonition. But this seems very harsh with no clear pointer as to when it is a good option.

Reply:
Hm. I could repeat what is on line 41 down here just as an emphasis...

```suggestion
   MinIO Object Transition supports cost-saving strategies around tiering older objects from the faster and typically more expensive "hot" tier to slower, less expensive "warm/cold" tiers.
   Object Transition does **not** provide backup and recovery functionality such as restoring data lost on the hot tier using the warm or cold tier.
```

Reply:
You need '...**to** less expensive "warm/cold" tiers.'
But I think this works.

Reply:
Github didn't put my comment where I expected. Anyway, I like your tweak, but you're missing a `to` after the last comma in line 65.